transit_odp/timetables/extract.py

```diff
@@ -87,6 +95,24 @@ def extract(self) -> ExtractedData:
         jp_sections, timing_links = self.extract_journey_pattern_sections()
         logger.debug("Finished extracting journey_patterns_sections")
 
+        vehicle_journeys = pd.DataFrame()
+        if is_timetable_visualiser_active:
+            # Extract VehicleJourneys
+            logger.debug("Extracting vehicle_journeys")
+            vehicle_journeys = self.extract_vehicle_journeys()
+            logger.debug("Finished extracting vehicle_journeys")
+
+        serviced_organisations = pd.DataFrame()
+        operating_profiles = pd.DataFrame()
+        if is_timetable_visualiser_active:
+            # Extract ServicedOrganisations
+            logger.debug("Extracting serviced_organisations")
+            (
+                serviced_organisations,
+                operating_profiles,
+            ) = self.extract_serviced_organisations()
+            logger.debug("Finished extracting serviced_organisations")
```

Comment:
can this block be moved inside the if condition above so that we have one if condition instead of two?

Reply:
Yeah, should be easier to manage. Changed this.